src/link.zig

```diff
@@ -117,6 +117,8 @@ pub const Options = struct {
     version: ?std.builtin.Version,
     libc_installation: ?*const LibCInstallation,
 
+    wasi_exec_model: ?Compilation.WasiExecModel = null,
```

Comment:
Could you add a sentence or two explaining what this option holds?

Reply:
https://github.com/ziglang/zig/pull/8978/commits/cd610975f2c401ec413787c04c2c1c8a34e54804 fixed!